File: Basic/Java/src/samples/QiView.java
package samples;


public class QiView {

    private String Id = "";
    private String Name = "";
    private String Description = "";
    private String SourceTypeId = "";
    private String TargetTypeId = "";
    private QiViewProperty[] Properties = new QiViewProperty[0];

    public String getId() {
        return Id;
    }

    public void setId(String id) {
        this.Id = id;
    }

    public String getName() {
        return Name;
    }

    public void setName(String name) {
        this.Name = name;
    }

    public String getDescription() {
        return Description;
    }

    public void setDescription(String description) {
        this.Description = description;
    }

    public String getSourceTypeId() {
        return this.SourceTypeId;
    }

    public void setSourceTypeId(String sourceTypeId) {
        this.SourceTypeId = sourceTypeId;
    }
    
    public String getTargetTypeId() {
        return TargetTypeId;
    }

    public void setTargetTypeId(String targetTypeId) {
        this.TargetTypeId = targetTypeId;
    }

    public QiViewProperty[] getProperties() {
        return Properties;
    }

    public void setProperties(QiViewProperty[] properties) {
        this.Properties = properties;
    }
}

File: Basic/DotNet/QiRestApiCore/QiView.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace QiRestApiCore
{
    public class QiView
    {
        public string Id
        {
            get;
            set;
        }

        public string Name
        {
            get;
            set;
        }

        public string Description
        {
            get;
            set;
        }

        public string SourceTypeId
        {
            get;
            set;
        }

        public string TargetTypeId
        {
            get;
            set;
        }

        public IList<QiViewProperty> Properties
        {
            get;
            set;
        }
    }
}
